syntax_checkers/yang/pyang.vim

```diff
@@ -0,0 +1,36 @@
+"============================================================================
+"File:        pyang.vim
+"Description: Syntax checking plugin for syntastic.vim
+"Authors:     joshua.downer@gmail.com
+"
+"============================================================================
+
+if exists('g:loaded_syntastic_yang_pyang_checker')
+    finish
+endif
+let g:loaded_syntastic_yang_pyang_checker = 1
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+function! SyntaxCheckers_yang_pyang_GetLocList() dict
+    let makeprg = self.makeprgBuild({})
+
+    let errorformat = '%f:%l:%m'
+
+    return SyntasticMake({
+        \ 'makeprg': makeprg,
+        \ 'errorformat': errorformat,
+        \ 'postprocess': ['filterForeignErrors'] })
+endfunction
+
+runtime! syntax_checkers/yang/pyang.vim
```

Comment:
This was taken from the file that I based the script on. It is not required so I have removed it.